config/base.py

```diff
@@ -140,6 +140,9 @@ def get_postgres_version():
     '127.0.0.1'
 ]
 
+ALLOWED_CIDR_NETS = [
```

Comment:
I assume this is used in the allow_cidr module. This change would only allow connections from the hard-coded subnet defined above. This won't work for production use.

Reply:
I intended to revert these changes, actually. It occurred to me after the fact that we could just use ngrok to rewrite the headers. Thanks for catching this!